packages/api-client/index.ts

```diff
@@ -15,12 +15,12 @@ import {
   GetQueueResponse,
   GetReleaseNotesResponse,
   GroupQuestionsParams,
-  GetSelfEnrollResponse,
+  //GetSelfEnrollResponse,
```

Comment:
I'm guessing this being commented out is why Vercel/Cypress/linting/tests are failing on self enrollment stuff? Why was this commented out? Same for the TACheckinTimesResponse

Reply:
I think it was a linting issue once we commented out the previous registration form